jme3-desktop/src/main/java/com/jme3/input/awt/AwtMouseInput.java

```diff
@@ -304,7 +306,7 @@ public void mouseMoved(MouseEvent awtEvt) {
             }
             lastKnownLocation.x = awtEvt.getX();
             lastKnownLocation.y = awtEvt.getY();
-
+            
```

Comment:
same accidental whitespace here?

Reply:
It was probably a couple of accidental keystrokes (I'm working on it).